packages/buttons/src/views/Anchor.js

```diff
@@ -34,30 +35,56 @@ const StyledAnchor = styled.a.attrs({
   ${props => retrieveTheme(COMPONENT_ID, props)};
 `;
 
+const StyledNewWindowIcon = styled(NewWindowIcon)`
+  margin-bottom: -0.085em;
+  padding-left: 0.25em;
+  width: 0.85em;
+  height: 0.85em;
+
+  ${props =>
+    isRtl(props) &&
+    `
+    transform: scaleX(-1);
```

Comment:
This approach to RTL is not reliable. Consider applying `dir: rtl` to the SVG or use `background` for left/right positioning.

Reply:
The background approach didn't work due to our use of `currentColor` in the SVGs.  I've added a new approach.